stdlib/public/core/Sequence.swift

```diff
@@ -877,6 +877,13 @@ extension Sequence {
   public func filter(
     _ isIncluded: (Element) throws -> Bool
   ) rethrows -> [Element] {
+    return try _filter(isIncluded)
```

Comment:
Right. I was just questioning whether this is the right thing. I think it is for those two types specifically.